### DISCUSSION TITLE
Hitdefguardflag trigger

### DISCUSSION THREAD

**Discussion by Kiwi1444** [ikemen-engine/Ikemen-GO#1544] [Feature Request] [answered]
### Is your feature request related to a problem? Please describe.

Today recognition of unguardable attacks or right state to guard (high or low) is a complex problem for AI. With high chance to fail

### Describe the solution you'd like

Trigger "Hitdefguardflag" will be very handy for that problem. Usage like that: enemynear,hitdefguardflag=(hit flag)
hit flags are: "H" for "high", "L" for "low", "A" for air. "M" for mid, like in hitdef contoller and new flag "U" = no flag (unguardable)

### Search terms

guardflag, hitdef

**Comment by potsmugen** [comment]
The same could be said about hitflags. Maybe there could be a new larger trigger like HitdefVar() or something like that where things like this could be checked.

**Comment by potsmugen** [comment, marked as answer]
Implemented in `HitDefVar` trigger.
